packages/core/src/extensions/common-api/app.ts

```diff
@@ -53,6 +53,10 @@ export const App = {
 
     return di.inject(isLinuxInjectable);
   },
-  slackUrl,
+  /**
+   * @deprecated Slack link is the same as the forums link
+   */
+  slackUrl: forumsUrl,
```

Comment:
I wonder if you should leave it pointing to slack? Changing it to forums does more than deprecate.

Reply:
My first thought was to make this just `""`. Since we don't want people going to slack anymore